analytics/__main__.py

```diff
@@ -5,6 +5,7 @@
 from analytics.metrics.exporter import start_metrics_server
 from analytics.processing.filters import EmgProcessor
 from analytics.adc.visualization import EmgVisualizer
+from analytics.adc.visualization import CalibrationVisualizer
```

Comment:
minor: cleaner import
```suggestion
from analytics.adc.visualization import CalibrationVisualizer, EmgVisualizer
```